menas/src/main/scala/za/co/absa/enceladus/menas/repositories/RunMongoRepository.scala

```diff
@@ -53,6 +55,51 @@ class RunMongoRepository @Autowired()(mongoDb: MongoDatabase)
     excludeId()
   ))
 
+  private def getTodaysFilter() = {
+    val date = LocalDate.now().format(DateTimeFormatter.ofPattern("dd-MM-yyyy"))
```

Comment:
This gives you local date where Menas is deployed right? Is the system-wide timezone set before this or used somehow?

Reply:
Uses OS settings.

Reply:
But that is bad, isn't it? OS settings are who knows what and we have everything in Run model in UTC.

Reply:
Created #541